languages/en.ini

```diff
@@ -229,6 +230,7 @@ citation_volume_abbrev = "Vol."
 Cite this = "Cite this"
 City = "City"
 Clear = "Clear"
+clear_feedback_filter = "Clear filter"
```

Comment:
Is there a strong reason why this text is "Clear filter" instead of "Clear Filter"? If we don't mind using the capitalized version, we could probably use the language console tools to copy the existing clear_tag_filter translation to clear_feedback_filter, and then we won't have to re-translate that string into all the various languages. I can run this command and commit the changes if you like!

Reply:
No reason, I've changed it copy the translation to all language files